### PR TITLE
[DEV-11322] Warn users if CTA is unsupported in their current theme

### PR DESCRIPTION
**Pull request by fgyimah**

### CODE REVIEW

File: packages/slate-editor/src/extensions/button-block/components/ButtonBlockMenu.tsx [hunk 1]
```diff
@@ -45,6 +46,11 @@ interface Props {
     onRemove: () => void;
     value: FormState;
     withNewTabOption: boolean;
+    infoText?: {
+        text: string;
+        actionLabel?: string;
+        action?: () => void;
+    };
```
Comment (e1himself) [inline packages/slate-editor/src/extensions/button-block/components/ButtonBlockMenu.tsx:53]: I have a bad feeling about this interface. I think it's too much tailored to the use-case to the point it's almost hard-coded.

How about changing it to something more generic:

```ts
info: Array<string | { text: string, href: string } | { text: string, onClick: () => void }>
```

And then pass `info` as:

```ts
info={[
    "Your current theme doesn't support these options. ", 
    { text: 'Go to themes', onClick: ... },
]} 
```

what do you think?

Comment (fgyimah) [reply packages/slate-editor/src/extensions/button-block/components/ButtonBlockMenu.tsx:53]: I agree with this, making the update

Comment (fgyimah) [reply packages/slate-editor/src/extensions/button-block/components/ButtonBlockMenu.tsx]: Updated

**Review by e1himself — APPROVED** [review]

**Pull request merged**